docs/installation.md

```diff
@@ -46,6 +46,8 @@ Install Bref in your project using [Composer](https://getcomposer.org/):
 composer require bref/bref
 ```
 
+> Make sure that you have installed a version greater than or equal to 0.5.* of Bref, to make use of the serverless plugin. Older version of Bref won't work with the serverless plugin.
```

Comment:
I think mentioning the serveless plugin is too early in the documentation (readers will not know what it means).

We can probably keep it simpler, "Make sure that you have installed a version greater than or equal to 0.5.* of Bref" is perfect on its own!